course/models.py

```diff
@@ -758,7 +758,8 @@ def get_late_submission_point_worth(self):
         Returns the percentage (0-100) that late submission points are worth.
         """
         point_worth = 0
-        if self.late_submissions_allowed:
+        if self.late_submission_penalty is not None:
```

Comment:
Why did you make this change? Late submissions are not accepted at all if `self.late_submissions_allowed` is `False`.

Reply:
I don't think `self.late_submission_penalty` could ever be `None` since the model field does not have null=True nor blank=True.